src/components/Button/_button.scss

```diff
@@ -60,6 +60,21 @@
     border-color: transparent;
   }
 
+  &--icon {
+    background-color: unset;
+    border: unset;
+    min-width: unset;
+
+    & > svg {
+      margin-inline-end: unset;
+      width: unset;
+    }
+
+    &:hover {
```

Comment:
hmm, I actually think the icon maybe should have a hover state and definitely a pointer to indicate that it's actionable.  The icon button doesn't really seem to be working for me in our stories